package.xml

```diff
@@ -8,10 +8,17 @@
 
   <license>MIT</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>mpi_cmake_modules</build_depend>
```

Comment:
buildtool_depend?

Reply:
Isn't this a normal build_depend?  I never fully understood this buildtool thing, though.